client/posts/embed.ts

```diff
@@ -7,7 +7,7 @@ type OEmbedDoc = {
 }
 
 // Types of different embeds by provider
-enum provider { Youtube, SoundCloud, Vimeo }
+enum provider { Youtube, Hooktube, SoundCloud, Vimeo }
```

Comment:
Could you append to the enum instead to break running applications less?

Reply:
Now that I look at that, you're right. The datatype stuff on the bottom makes sense too now.
Sorry about that, I'll fix it.